lib/open_project/gitolite/load_gitolite_hooks.rb

```diff
@@ -64,6 +64,13 @@
   end
 end
 
+# Gitolite hooks can be found in Redmine root dir or in plugin root dir
```

Comment:
Please remove the commented code and put it somewhere else (Readme, initializer, etc.).